site/en/blog/private-prefetch-proxy/index.md

```diff
@@ -0,0 +1,137 @@
+or [how referrer websites can help their users](#referrers) achieve their goals by speeding up cross-site navigations.
+
+## How private prefetch proxy works {: #how }
+
+### Secure communication channel
+
+This feature uses a [CONNECT](https://tools.ietf.org/html/rfc7231#section-4.3.6) proxy to establish a secure communication channel between Chrome and the server hosting the content to be prefetched. This secure communication channel prevents the proxy from inspecting any data transfer. Notably, while the private prefetch proxy necessarily sees the host name in order to establish a secure communication channel, it does not see the full URLs, nor the resources themselves.
+
+<figure>
+{% Img src="image/kheDArv5csY6rvQUJDbWRscckLr1/UF6dpd6QvSzGXXlM9YWE.gif", alt="Animation showing flow of data through proxy.", width="600", height="365" %}
+  <figcaption>Prefetching websites via a CONNECT proxy prevents leaking user information.</figcaption>
+</figure>
+
+In addition, because the secure communication channel is encrypted end-to-end, intermediaries can neither observe the host names, nor the content of the prefetched sites. Finally, the proxy inherently prevents the destination server from seeing the user's IP address.
+
+### Preventing user identification
+
+Beyond the network aspects detailed earlier, we also need to prevent servers from identifying a user at prefetch time, via information previously stored on their device. To that end, Chrome currently restricts the usage of Private Prefetch Proxy to websites for which the user has no cookies or other local state. Here are the restrictions for prefetch requests made via the private prefetch proxy:
+
+- **Cookies:** Prefetch requests are not allowed to carry cookies.
+  - If there is a cookie for a resource, Chrome will make an uncredentialed fetch but will not use the response (see later [Caching](#caching) section). 
+  - Although responses to a prefetch request can include cookies, these cookies will only be saved if the user navigates to the prefetched page.
+- **Fingerprinting:** Other surfaces which could be used for fingerprinting are also adjusted. For example, the User-Agent header sent by the prefetch proxy only carries limited information.
+
+In the future, we hope to expand private prefetch proxy to links with cookies or local state while maintaining the same privacy characteristics. See the [What's next](#next) section for more details.
+
+### Caching {: #caching }
+
+Chrome will prefetch resources even if they are already in the cache, but they will not carry any conditional headers such as Etags or If-Modified-Since (these contain server-set values which could be used for tracking even without cookies). This prefetching is done to prevent leaking a client's cache state to the prefetched website. In addition, Chrome will only commit a prefetched resource to the cache if the user decides to navigate to the prefetched website.
```

Comment:
```suggestion
Chrome will prefetch resources even if they are already in the cache, but they will not carry any conditional headers such as `ETag` or `If-Modified-Since` (these contain server-set values which could be used for tracking even without cookies). This prefetching is done to prevent leaking a client's cache state to the prefetched website. In addition, Chrome will only commit a prefetched resource to the cache if the user decides to navigate to the prefetched website.
```